backend/django_core/static_src/components/minor/Comment/Index.svelte

```diff
@@ -74,11 +75,11 @@
     });
 </script>
 
-{#if loading_state === "loading"}
-    Loading..
+{#if loading_state === "loaded"}
+    <CommentSkeleton />
 {:else if loading_state === "error"}
     Something is wrong Error : {@html error}
-{:else if loading_state === "loaded"}
+{:else if loading_state === "loading"}
```

Comment:
@baseplate-admin revert these states once you are okay :\

Reply:
ohh. makes sense